packages/util/tests/dummy/app/templates/application.hbs

```diff
@@ -1,3 +1,3 @@
-<h2 id="title">Welcome to Ember</h2>
```

Comment:
I feel like this shouldn't be changed 🤔 because it's not the same as a default output

Reply:
resolved!

Reply:
resolved for real now